docs/advanced/transform-content.md

```diff
@@ -26,15 +46,44 @@ const mdx = require('remark-mdx');
 })();
 ```
 
-As the comment in the code above points out, you can use any of the [remark plugins][remark-plugins], or even write your own, to interpolate MDX content.  The remark-mdx library is a parsing extension to enhance the Markdown [AST][ast] to understand MDX (resulting in MDXAST), giving you access and insight to things like imports and exports.
+Now, running `node example.js` yields:
+
+```js
+{
+  type: "root",
+  children: [
+    {
+      type: "import",
+      value: "import Component from 'Component';",
+      position: [Object]
+    },
+    {
+      type: "export",
+      value: "export const meta = {\n    prop: 'value'\n};",
+      position: [Object]
+    },
+    { type: "heading", depth: 1, children: [Array], position: [Object] },
+    { type: "paragraph", children: [Array], position: [Object] },
+    { type: "jsx", value: "<Component />", position: [Object] }
+  ],
+  position: {
+    start: { line: 1, column: 1, offset: 0 },
+    end: { line: 12, column: 1, offset: 113 }
+  }
+}
+```
+
+From here, you can continue to [write your own plugin][writing-a-plugin] or use any of the [remark plugins][remark-plugins] to interpolate MDX content.
```

Comment:
Is `interpolate` the correct term here? English isn’t my primary language, so maybe it is, but it seems a bit jargon-y to me. Maybe just drop the ` to interpolate MDX content` part?

Reply:
@wooorm I went with `parse or manipulate MDX content` instead.

Reply:
Good good!